src/python/espressomd/thermostat.pyx

```diff
@@ -330,39 +343,55 @@ cdef class Thermostat(object):
 
     IF LB_GPU or LB:
         @AssertThermostatType(THERMO_LB)
-        def set_lb(self, kT=None, act_on_virtual=True):
+        def set_lb(
+            self,
+            seed=None,
+            act_on_virtual=True,
+            LB_fluid=None,
+                friction=0.0):
```

Comment:
If think the naming should be consistent with the Langevin thermostat, where the friction parameter is called gamma. Also the docstring for `friction` is missing.